CHANGELOG.md

```diff
@@ -7,10 +7,6 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
-## [2.74.0] - 2019-11-11
```

Comment:
Bring this back and add a YANKED after the date

![image](https://user-images.githubusercontent.com/284515/68613770-44923d80-049e-11ea-929e-384dd2696f1d.png)

Reply:
Niice!